mempool/clist_mempool_test.go

```diff
@@ -271,14 +271,15 @@ func TestMempoolUpdateDoesNotPanicWhenApplicationMissedTx(t *testing.T) {
 	mockClient.On("SetLogger", mock.Anything)
 	mockClient.On("Error").Return(nil).Times(4)
 	mockClient.On("SetResponseCallback", mock.MatchedBy(func(cb abciclient.Callback) bool { callback = cb; return true }))
+	mockClient.On("CheckTxAsync", mock.Anything, mock.Anything).Return(nil, nil)
+	mockClient.On("Flush", mock.Anything).Return(nil)
```

Comment:
This test passes in `main`. Curious on why we have to mock `Flush` here.

Reply:
`Flush` is called when sending the requests to the app, so if it's not mocked the mock client will complain.